docs/developing/managing-focus.md

```diff
@@ -236,3 +238,44 @@ export default Home;
 </TabItem>
 </Tabs>
 ````
+
+## Assistive Technology Focus Management
+
+By default, Single Page Applications have no built-in way of informing screen readers that the active view has changed in a browser or webview. This means that users who rely on assistive technology do not always know if a navigation event occurred.
+
+Developers who enable the [focusManagerPriority config](./config#ionicconfig) can delegate focus management to Ionic during page transitions. When enabled, Ionic will move focus to the correct element as specified in the config option. This will inform screen readers that a navigation event occurred.
+
+### Types
+
+```typescript
+type FocusManagerPriority = 'content' | 'heading' | 'banner';
+```
+
+### Content Types
+
+The following table explains what each content type represents:
+
+| Type      | Description                   | Ionic Component           | Semantic HTML Equivalent                                                     | Landmark Equivalent                                                                                                                                                                                                   |
+| --------- | ----------------------------- | ------------------------- | ---------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `content` | The main portion of the view. | [Content](../api/content) | [`main`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main)     | [`role="main"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role)                                                                                                                      |
+| `heading` | The title of the view.        | [Title](../api/title)     | [`h1`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/h1)         | [`role="heading"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role) with [`aria-level="1"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level) |
+| `banner`  | The header of the view.       | [Header](../api/header)   | [`header`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) | [`role="banner"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role)                                                                                                                  |
+
+:::important
+Developers should assign `role="heading"` and `aria-level="1"` to the primary [Title](../api/title) on each view. Since multiple [Title](../api/title) components can be used in a single view, Ionic does not automatically assign these attributes.
```

Comment:
We already call this out on https://ionicframework.com/docs/api/title#headings

Reply:
Should we link to the `#headings` section here then?

Reply:
added in https://github.com/ionic-team/ionic-docs/pull/3627/commits/611dee9e2ed7a4466ba0bdf3cb1d1c70ff54e197

Reply:
Oh I meant the opposite. Meaning this:

```suggestion
Developers should assign `role="heading"` and `aria-level="1"` to the primary [Title](../api/title#headings) on each view. Since multiple [Title](../api/title#headings) components can be used in a single view, Ionic does not automatically assign these attributes.
```